### PR TITLE
Windowcontrols

### PR DESCRIPTION
**Pull request by gordonbrander**
Caution: this _must_ be landed on top of https://github.com/mozilla/browser.html/pull/681.
- [x] Update window glyph sprite
- [x] Add style for unfocused window state
- [x] Add hover state with glyphs
- [x] Add Pointer model and actions (for representing hover)
- [x] Add WindowShell model and actions
- [x] Update initial state of model
- [x] Bump session version number

Additional details:
- Need to add option state (but first we need to implement expand-to-fit)
- Need to add full-screen variant (this may have to be a native cocoa control)

### CODE REVIEW

File: src/browser/synthesis-ui.js [hunk 1]
```diff
@@ -276,13 +281,15 @@
     action instanceof Suggestions.PageResult ?
       updateSuggestions(state, action) :
 
-    (
-      action instanceof Focusable.Focused ||
-      action instanceof Focusable.Focus ||
-      action instanceof Focusable.Blured ||
-      action instanceof Focusable.Blur
-    ) ?
-      state.set('shell', Focusable.update(state.shell, action)) :
```
Comment (Gozala) [inline src/browser/synthesis-ui.js:285]: Did you meant to remove these lines ?

Comment (Gozala) [inline src/browser/synthesis-ui.js:285]: Never mind I see you handle them below

**Comment by Gozala** [comment]
P.S.: I think you could greatly simplify stuff if you aligned hovered button glimpse on the right to the non hovered ones.

**Comment by gordonbrander** [comment]
@Gozala sorry, I'm not sure what that means

**Pull request merged**